### PR TITLE
EMDA 177

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Authors of old qenes from EMDA 177.

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
Please have a look :)

File: new/PRS14471AkalaKrestos.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <change who="DN" when="2024-06-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ʾAkala Krǝstos</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Akāla Krǝstos</persName>
+                    <t:faith xmlns:t="http://www.tei-c.org/ns/1.0" type="EOTC"/>
```
Comment (eu-genia) [inline new/PRS14471AkalaKrestos.xml:50]: remove t: and xmlns

File: new/PRS14472WaldaKrestos.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <change who="DN" when="2024-06-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወልደ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda Krǝstos</persName>
+                    <t:faith xmlns:t="http://www.tei-c.org/ns/1.0" type="EOTC"/>
```
Comment (eu-genia) [inline new/PRS14472WaldaKrestos.xml:50]: remove t:
should be just `<faith type="EOTC"/>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS14472WaldaKrestos.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS14472WaldaKrestos.xml:50]: ok

**Pull request merged**